pkg/cmd/cli/install/install.go

```diff
@@ -118,6 +120,8 @@ func (o *InstallOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.Features, "features", o.Features, "Comma separated list of Velero feature flags to be set on the Velero deployment and the node-agent daemonset, if node-agent is enabled")
 	flags.BoolVar(&o.DefaultVolumesToFsBackup, "default-volumes-to-fs-backup", o.DefaultVolumesToFsBackup, "Bool flag to configure Velero server to use pod volume file system backup by default for all volumes on all backups. Optional.")
 	flags.StringVar(&o.UploaderType, "uploader-type", o.UploaderType, fmt.Sprintf("The type of uploader to transfer the data of pod volumes, the supported values are '%s', '%s'", uploader.ResticType, uploader.KopiaType))
+	flags.BoolVar(&o.Uninstall, "uninstall", o.Uninstall, "Uninstall Velero from the cluster. Optional.")
```

Comment:
Why do we add a `--uninstall` flag under `velero install` instead of under `velero uninstall`?
It is hard to understand if the command is like "velero install --uninstall".

Reply:
Because uninstall command do not have the install options.

If we want this functionality in uninstall cmd then the installOptions fields will have to be included with uninstall cmd.

This makes it clear that it'll only work if you know how you installed it.

Perhaps even lookup your bash history and just add --uninstall

Reply:
> Because uninstall command do not have the install options

I thinks this is a problem of the code organization, we have multiple ways to solve it. i.e., Add a new UninstallOption struct.

> This makes it clear that it'll only work if you know how you installed it.

Maybe, but this will confuse others who don't know the details.

Anyway, as mentioned about, I think even the flag `--preserve-uninstall-namespace` is unnecessary if we want to fix requirement 1 only.